Plumbing DeleteSite operation casts interface to wrong type, panics

From calling `operations.DeleteSite`:

```
panic: interface conversion: interface {} is *operations.DeleteSiteDefault, not *operations.DeleteSiteOK

goroutine 205 [running]:
github.com/netlify/open-api/go/plumbing/operations.(*Client).DeleteSite(0xc42032d7e0, 0xc42039b110, 0x1db43e0, 0xc4203d75d0, 0x30, 0x1894dc0, 0x1930501)
	/Users/mitchellh/code/go/src/github.com/netlify/open-api/go/plumbing/operations/operations_client.go:400 +0x56e
```

--- a/go/plumbing/operations/delete_site_responses.go
+++ b/go/plumbing/operations/delete_site_responses.go
@@ -25,8 +25,8 @@ type DeleteSiteReader struct {
 func (o *DeleteSiteReader) ReadResponse(response runtime.ClientResponse, consumer runtime.Consumer) (interface{}, error) {
 	switch response.Code() {
 
-	case 200:
-		result := NewDeleteSiteOK()
+	case 204:
+		result := NewDeleteSiteNoContent()
 		if err := result.readResponse(response, consumer, o.formats); err != nil {
 			return nil, err
 		}
@@ -44,23 +44,23 @@ func (o *DeleteSiteReader) ReadResponse(response runtime.ClientResponse, consume
 	}
 }
 
-// NewDeleteSiteOK creates a DeleteSiteOK with default headers values
-func NewDeleteSiteOK() *DeleteSiteOK {
-	return &DeleteSiteOK{}
+// NewDeleteSiteNoContent creates a DeleteSiteNoContent with default headers values
+func NewDeleteSiteNoContent() *DeleteSiteNoContent {
+	return &DeleteSiteNoContent{}
 }
 
-/*DeleteSiteOK handles this case with default header values.
+/*DeleteSiteNoContent handles this case with default header values.
 
 OK
 */
-type DeleteSiteOK struct {
+type DeleteSiteNoContent struct {
 }
 
-func (o *DeleteSiteOK) Error() string {
-	return fmt.Sprintf("[DELETE /sites/{site_id}][%d] deleteSiteOK ", 200)
+func (o *DeleteSiteNoContent) Error() string {
+	return fmt.Sprintf("[DELETE /sites/{site_id}][%d] deleteSiteNoContent ", 204)
 }
 
-func (o *DeleteSiteOK) readResponse(response runtime.ClientResponse, consumer runtime.Consumer, formats strfmt.Registry) error {
+func (o *DeleteSiteNoContent) readResponse(response runtime.ClientResponse, consumer runtime.Consumer, formats strfmt.Registry) error {
 
 	return nil
 }

--- a/go/plumbing/operations/delete_site_responses.go
+++ b/go/plumbing/operations/delete_site_responses.go
@@ -51,7 +51,7 @@ func NewDeleteSiteNoContent() *DeleteSiteNoContent {
 
 /*DeleteSiteNoContent handles this case with default header values.
 
-OK
+Deleted
 */
 type DeleteSiteNoContent struct {
 }

--- a/go/plumbing/operations/operations_client.go
+++ b/go/plumbing/operations/operations_client.go
@@ -375,7 +375,7 @@ func (a *Client) DeleteHookBySiteID(params *DeleteHookBySiteIDParams, authInfo r
 /*
 DeleteSite delete site API
 */
-func (a *Client) DeleteSite(params *DeleteSiteParams, authInfo runtime.ClientAuthInfoWriter) (*DeleteSiteOK, error) {
+func (a *Client) DeleteSite(params *DeleteSiteParams, authInfo runtime.ClientAuthInfoWriter) (*DeleteSiteNoContent, error) {
 	// TODO: Validate the params before sending
 	if params == nil {
 		params = NewDeleteSiteParams()
@@ -397,7 +397,7 @@ func (a *Client) DeleteSite(params *DeleteSiteParams, authInfo runtime.ClientAut
 	if err != nil {
 		return nil, err
 	}
-	return result.(*DeleteSiteOK), nil
+	return result.(*DeleteSiteNoContent), nil
 
 }
 

--- a/swagger.yml
+++ b/swagger.yml
@@ -102,7 +102,7 @@ paths:
     delete:
       operationId: deleteSite
       responses:
-        '200':
+        '204':
           description: OK
         default:
           description: error

--- a/swagger.yml
+++ b/swagger.yml
@@ -103,7 +103,7 @@ paths:
       operationId: deleteSite
       responses:
         '204':
-          description: OK
+          description: Deleted
         default:
           description: error
           schema:

--- a/ui/swagger.json
+++ b/ui/swagger.json
@@ -788,7 +788,7 @@
         "operationId": "deleteSite",
         "responses": {
           "204": {
-            "description": "OK"
+            "description": "Deleted"
           },
           "default": {
             "description": "error",

--- a/ui/swagger.json
+++ b/ui/swagger.json
@@ -787,7 +787,7 @@
       "delete": {
         "operationId": "deleteSite",
         "responses": {
-          "200": {
+          "204": {
             "description": "OK"
           },
           "default": {

